MdeModulePkg/Include/Guid/VariableRuntimeCacheContext.h

```diff
@@ -0,0 +1,38 @@
+/** @file
+  This file defines a VARIABLE_RUNTIME_CACHE_CONTEXT_HOB.
+  It provides the variable runtime cache context.
+
+  Copyright (c) 2024, Intel Corporation. All rights reserved.<BR>
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef VARIABLE_RUNTIME_CACHE_CONTEXT_H_
+#define VARIABLE_RUNTIME_CACHE_CONTEXT_H_
+
+#include <PiPei.h>
+#include <Guid\SmmVariableCommon.h>
+
+#define VARIABLE_RUNTIME_CACHE_CONTEXT_HOB_REVISION  1
+
+#define VARIABLE_RUNTIME_CACHE_CONTEXT_GUID \
+  { \
+    0x0f472f7d, 0x6713, 0x4915, {0x96, 0x14, 0x5d, 0xda, 0x28, 0x40, 0x10, 0x56}  \
+  }
+
+typedef struct {
+  UINT64    ReadLockPtr;

```

Comment:
can you add more comments to each field in the structure?
At least including the buffer size each *Ptr points to.
Better to describe how each field is used in a cache logic.